product_variants_no_automatic_creation/views/product_view.xml

```diff
@@ -17,18 +17,54 @@
             <field name="model">product.template</field>
             <field name="inherit_id" ref="product.product_template_only_form_view" />
             <field name="arch" type="xml">
-                <button string="Variant Prices" position="attributes">
+            <xpath expr="//button[@name='103']" position="attributes">
```

Comment:
Ok so if i understand right, these IDs are generated upon installation and
can vary per install.
so the 103 name is the name from my current install and therefore working
as intended.
And for that it was looking for the text string

2015-12-14 12:51 GMT-05:00 Pedro M. Baeza <notifications@github.com>:

> In product_variants_no_automatic_creation/views/product_view.xml
> <https://github.com/odoomrp/odoomrp-wip/pull/1098#discussion_r47531221>:
>
> > @@ -17,18 +17,54 @@
> >              <field name="model">product.template</field>
> >              <field name="inherit_id" ref="product.product_template_only_form_view" />
> >              <field name="arch" type="xml">
> > -                <button string="Variant Prices" position="attributes">
> > +            <xpath expr="//button[@name='103']" position="attributes">
>
> But this is because it's replaced in execution time by its current XML-ID.
> Here you can see the correct expression:
> https://github.com/odoo/odoo/blob/9.0/addons/product/product_view.xml#L77
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/odoomrp/odoomrp-wip/pull/1098/files#r47531221>.
>